Resources/Private/BackendModule/src/index.tsx

```diff
@@ -45,28 +48,51 @@ document.addEventListener('DOMContentLoaded', async() => {
     const externalService = ExternalService.getInstance()
     externalService.configure(configuration.apiDomain, configuration.apiKey, configuration.userInterfaceLanguage)
 
+    let availableNodeTypeFilters = null
+    if (scope === 'focusKeywordGeneratorModule') {
+        availableNodeTypeFilters = {}
+        const nodeTypeSchema: Partial<{nodeTypes: object}> = await backend.getNodeTypeSchema()
+        let availableNodeTypes = nodeTypeSchema.nodeTypes
+        Object.keys(availableNodeTypes).map(nodeType => {
+            const nodeTypeDefinition: {superTypes: object, ui: {label: string}} = availableNodeTypes[nodeType]
+            if (nodeTypeDefinition?.superTypes?.hasOwnProperty('NEOSidekick.AiAssistant:Mixin.AiPageBriefing')) {
+                availableNodeTypeFilters[nodeType] = translationService.translate(nodeTypeDefinition.ui.label, nodeTypeDefinition.ui.label)
+            }
+        })
+    }
+
+    // Backend Message Identifier
+    let backendMessageIdentifier
+    switch(scope) {
+        case 'altTextGeneratorModule':
+            backendMessageIdentifier = 'bulk-image-generation'; break;
```

Comment:
Gibts einen grund warum backendMessageIdentifier und scope nicht einfach der gleiche Name is?